.github/workflows/tests.yaml

```diff
@@ -39,43 +43,46 @@ jobs:
     steps:
       # Ubuntu images start with 23GB available, and this adds 14GB more. For
       # comparison, MacOS images have >100GB free.
+      #
+      # Although we could delete more, if we run into a limit, not deleting
+      # everything provides a little flexibility to get space while trying
+      # to shrink the build.
```

Comment:
Should we keep this comment next to the action it refers to?

Reply:
Fixed